packages/frontend/src/utils/wallet.ts

```diff
@@ -113,6 +130,14 @@ export async function getKeyMeta(publicKey) {
 }
 
 export default class Wallet {
+    accountId;
```

Comment:
I think we need to add types to these class properties- they are currently just seen as `any`

Reply:
we can always come back to enhance this file in the future, but at least use a .ts extension with any is always better than use .js